packages/stacks-svelte/package.json

```diff
@@ -0,0 +1,43 @@
+{
+    "name": "@stackoverflow/stacks-svelte",
```

Comment:
I renamed the package from `@stackeng/stacks-svelte` to `@stackoverflow/stacks-svelte` so it matches the org name for stacks classic. We can update internal consumers to the new package in separate PR's.